app/views/content/landing/food-for-thought.md

```diff
@@ -6,7 +6,7 @@ content:
     - content/landing/food-for-thought/quotes
     - content/landing/food-for-thought/promo
 image: "static/images/content/hero-images/cake.png"
-colour: "yellow"
 layout: "layouts/minimal"
+colour: pink-blue
```

Comment:
@jenhadfield-dfe This is content for freshers so should reflect the on campus colours - so pink/yellow rather than pink/blue

Reply:
updated thanks